packages/ember-routing-htmlbars/lib/keywords/action.js

```diff
@@ -7,7 +7,7 @@ import { keyword } from 'htmlbars-runtime/hooks';
 import closureAction from 'ember-routing-htmlbars/keywords/closure-action';
 
 /**
-  The `{{action}}` helper provides a way to pass triggers for behavior (usually
+  The `{{action` helper provides a way to pass triggers for behavior (usually
```

Comment:
Hmm. I've seen this style in a number of places, but looking in the guides it might actually be the exception.

My understanding was that `{{action` (or whatever helper) without the closing double-curly was to make clear that `{{action` is just the beginning of the helper declaration, as opposed to something like `{{debugger}}`, which is by itself the whole declaration.

I'm trying to find a reference for this usage, but can't seem to at the moment. And looking through the guides and docs, it seems to go against the current convention.

This file used both versions before. I just think that we need to be consistent with one or the other. I don't really care either way.